gapis/service/path/path.proto

```diff
@@ -48,23 +48,24 @@ message Any {
     ImageInfo image_info = 20;
     MapIndex map_index = 21;
     Memory memory = 22;
-    Mesh mesh = 23;
-    Metrics metrics = 24;
-    Messages messages = 25;
-    MultiResourceData multi_resource_data = 26;
-    Parameter parameter = 27;
-    Report report = 28;
-    ResourceData resource_data = 29;
-    Resources resources = 30;
-    Result result = 31;
-    Slice slice = 32;
-    State state = 33;
-    StateTree state_tree = 34;
-    StateTreeNode state_tree_node = 35;
-    StateTreeNodeForPath state_tree_node_for_path = 36;
-    Stats stats = 37;
-    Thumbnail thumbnail = 38;
-    Type type = 39;
+    MemoryAsType memoryAsType = 23;
```

Comment:
So here we reused the tag id, but I guess it is ok since the the gapis and gapit are always build and delivered the same time.

Reply:
That is correct. If we decide in the future to decouple them, we will have to be more careful.